code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -273,9 +273,9 @@
 			else
 				playsound(loc, 'sound/weapons/pierce.ogg', 25, 1, -1)
 				if(!lying)				//CITADEL EDIT
-					Knockdown(100, TRUE, FALSE, 30, 25)
+					Knockdown(25, TRUE, FALSE, 30, 25)

```

Comment:
This isn't even the right arg to modify to change the hardstun time

Reply:
Then what is? KD should stun its a stun for batons I thought...